app/jobs/update_account_invoices.rb

```diff
@@ -37,9 +37,7 @@ def update(account_invoice)
       if billable_periods.any?
         oldest_enterprise = billable_periods.first.enterprise
         address = oldest_enterprise.address.dup
-        first, _space, last = (oldest_enterprise.contact || "").partition(' ')
         address.update_attributes(phone: oldest_enterprise.phone) if oldest_enterprise.phone.present?
-        address.update_attributes(firstname: first, lastname: last) if first.present? && last.present?
         account_invoice.order.update_attributes(bill_address: address, ship_address: address)
       end
 
```

Comment:
If we're adding a name to business invoices I think we should have a specific field which is clearly labelled as being a named person to whom enterprise invoices should be made out to, which in some cases would not be the person receiving notification emails. And in some cases people would want business invoices to be in the name of the company instead.